required/latex-lab/testfiles-table-pdftex/table-012-caption.tlg

```diff
@@ -13,61 +13,61 @@ Don't change this file in any respect.
 ==> (@arraycr) This row needs 1 additional cell(s)
 --longtable--> chunk row: 1 row: 2 column: 1
 ==> Inserting 1 additional cell(s) into previous row:
-==> current cell data: 2,1,1
+==> current cell data: 2,2,1
 ==> structure stored for row 2: 9,10
 ==> current cell data: 3,1,1
 ==> (@arraycr) This row needs 1 additional cell(s)
 --longtable--> chunk row: 2 row: 3 column: 1
 ==> Inserting 1 additional cell(s) into previous row:
-==> current cell data: 3,1,1
+==> current cell data: 3,2,1
 ==> structure stored for row 3: 12,13
```

Comment:
that is from the change in `\@@_add_missing_cells:` command. Previously it simply added a lot of empty TD structures to fill up the row but didn't adapt the column number. So all these empty cells had the same row/col number and it could happen that the multirow code suppressed them all instead of only the first one.  You can see that in the log previously there was 

~~~
==> current cell data: 3,1,1
...
==> current cell data: 3,1,1
~~~

so the same numbers for two different cells, and now it is 

~~~
==> current cell data: 3,1,1
...
==> current cell data: 3,2,1
~~~

Reply:
ah so a correct change :-)   should have deduced that from the diff as it indeed shows already there